src/moneypenny/errors.py

```diff
@@ -0,0 +1,45 @@
+"""Helper globals and errors for Moneypenny.
+"""
+
+namespace_file = "/var/run/secrets/kubernetes.io/serviceaccount/namespace"
```

Comment:
Probably don't need the namespace file since it's in the configuration object?

Reply:
done